utils/growpart/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2022 Eero Vuojolahti
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=growpart
+PKG_SOURCE_DATE:=2022-02-04
+PKG_SOURCE_VERSION:=4cc4950394b3db0243cb0211d6a394d629129a2a
+PKG_RELEASE:=1
```

Comment:
I would suggest to use autorelease

Reply:
What is that? "ctrl-f autorelease" shows exacty 0 hits on https://openwrt.org/docs/guide-developer/packages